frame_id:           "base_link"
obstacles:          "/sectorgrid"
targets:            "/user"
action_server:      "move_base"
obstacle_strength:  1.0
obstacle_decay:     1.0
obstacle_occupancy: 50.0
